src/main/java/homeworks/jose/secondAssigment/ClassB.java

```diff
@@ -0,0 +1,10 @@
+package homeworks.jose.secondAssigment;
+
+public class ClassB extends Father {
+    private Set<> e;
+
+    @Override
+    public void methodB() {
+        super.methodB();
```

Comment:
there is no need for an override (and therefore the method) since the behavior of both methods are the same